src/lib.rs

```diff
@@ -2771,6 +2775,23 @@ impl Build {
         })
     }
 
+    fn get_dwarf_version(&self) -> Option<u32> {
```

Comment:
I would argue that it would be more than appropriate to add a commentary stating that the version choice reflects rustc defaults as of version X through Y. Idea is that whenever defaults change, there would be a hint that corresponding adjustment would be needed.